pandas/core/generic.py

```diff
@@ -546,6 +546,52 @@ def swapaxes(self, axis1, axis2, copy=True):
     def pop(self, item):
         """
         Return item and drop from frame. Raise KeyError if not found.
+
+        Parameters
+        ----------
+        item : str
+            column label to be popped
```

Comment:
Can you capitalize the first word , so it reads like a full sentence? (although grammatically it isn't, but that is OK)